examples/solids/src/setuplibceed.c

```diff
@@ -475,6 +498,69 @@ PetscErrorCode SetupLibceedFineLevel(DM dm, DM dmEnergy, DM dmDiagnostic,
   data[fineLevel]->qfEnergy = qfEnergy;
   data[fineLevel]->opEnergy = opEnergy;
 
+  // ---------------------------------------------------------------------------
+  // Diagnostic value computation
+  // ---------------------------------------------------------------------------
+  // Create the QFunction and Operator that computes stress, strain energy density
```

Comment:
do you want to update this with:
```suggestion
  // Create the QFunction and Operator that computes pressure, strain energy density
```
?

Reply:
Good catch, I was going to put in stress. Really, the question is what all do we want as our diagnostic outputs for viz? It's now really easy to add different quantities to this PR.

Reply:
You can include the three invariants of the stress tensor.

Reply:
Added trace(E), trace(E^2), det(E)

Reply:
Can you make the third `J`? It's far more interpretable.

Reply:
Sure. How would that translate for linElas/hyperSS? Do the same values make sense with e?

Reply:
Yeah, it's `1 + tr(\epsilon)` in that case.

Reply:
Got it. Should be better now. Any other variables? Currently
- displacement x
- displacement y
- displacement z
- condensed pressure
- trace e/E
- trace e*e/E*E
- detJ
- strain energy density

(note, displacement needed for deforming to final configuration in Paraview)

Reply:
In principle, displacements could be a different field in DMPlex, and that might make it easier to handle as a vector in Paraview. I think your list is good for our current purposes.

Reply:
Right. I had considered that. I went with this since the DM setup and restriction setup is a bit simpler this way and warping by a subset of a vector isn't too bad in Paraview, but I can change it up if desired.